ksql-streams/src/main/java/io/confluent/ksql/execution/streams/materialization/ks/KsStateStore.java

```diff
@@ -72,9 +72,15 @@ LogicalSchema schema() {
     awaitRunning();
 
     try {
-
-      return kafkaStreams
-          .store(StoreQueryParams.fromNameAndType(stateStoreName, queryableStoreType));
+      if (ksqlConfig.getBoolean(KsqlConfig.KSQL_PULL_QUERIES_ENABLE_CONFIG)) {
```

Comment:
how is the KsqlRestConfig entry used

Reply:
I assume this is meant to be KSQL_QUERY_PULL_ENABLE_STANDBY_READS

Reply:
It's not

Reply:
Actually @AlanConfluent is right.. we should tie the standby routing to this, right?  not the pull query feature itself.

Reply:
Yes, the rebase overwrote my change. Fixed it now (again!)